docs/api-reference/geo-layers/imagery-layer.md

```diff
@@ -0,0 +1,179 @@
+<p class="badges">
+  <img src="https://img.shields.io/badge/lighting-yes-blue.svg?style=flat-square" alt="lighting" />
+</p>
+
+# ImageryLayer (Experimental)
+
+> From v8.9
+
+The `ImageryLayer` is a composite layer that connects with an image service that can render map images optimized for the current view. Instead of loading a detailed map image covering the entire globe, an image is rendered.
+
+To use this layer, an *image source* must be specified. Image sources are typically specified by supplying a URL to the `ImageryLayer` `data` property. See the section on image sources below for mor information.
+
+> Caveats: 
+> - The `ImageryLayer` currently does not work well with multiple views.
+> - The `ImageryLayer` currently does not work well with perspective views.
+> - The `ImageryLayer` has not been adapted to work with non-geospatial views such as the [OrthographicView](/docs/api-reference/core/orthographic-view.md) or the [OrbitView](/docs/api-reference/core/orbit-view.md).
+
+
+```typescript
+import DeckGL from '@deck.gl/react';
+import {BitmapLayer} from '@deck.gl/layers';
+import {ImageryLayer} from '@deck.gl/geo-layers';
+
+function App({viewState}) {
+  const layer = new ImageryLayer({
+    data: `https://ows.terrestris.de/osm/service`,
+    serviceType: 'wms',
+    layers: ['OSM-WMS']
+  });
+
+  return <DeckGL viewState={viewState} layers={[layer]} />;
+}
+```
+
+
+## Installation
+
+To install the dependencies from NPM:
+
+```bash
+npm install deck.gl
+# or
+npm install @deck.gl/core @deck.gl/layers @deck.gl/geo-layers
+```
+
+```typescript
+import {ImageryLayer} from '@deck.gl/geo-layers';
+new ImageryLayer({});
+```
+
+To use pre-bundled scripts:
+
+```html
+<script src="https://unpkg.com/deck.gl@^8.0.0/dist.min.js"></script>
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@^8.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/layers@^8.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/geo-layers@^8.0.0/dist.min.js"></script>
+```
+
+```typescript
+new deck.ImageryLayer({});
+```
+
+## Image Sources
+
+The `ImageryLayer` knows how to URLs for some well-known geospatial services such as WMS, however it is also possible to specify a custom URL template. 
```

Comment:
It's not clear what the purpose of this section is. Are you describing the features that this layer supports, or the features that a valid image source must implement?